.clj-kondo/status-im/utils/i18n.clj

```diff
@@ -0,0 +1,26 @@
+(ns utils.i18n
```

Comment:
Looking at the namespace change now @akatov, and the changes don't make much sense to me. Utility namespaces are supposed to mean they can be called/reused to build something else.

Originally, I added the `label` hook to a `core` namespace because this is part of our public hooks API so to speak. No other code should call `label`, except clj-kondo itself.

Could you please revert this namespace change?

Reply:
The intention was to match the namespace and name of the symbols that are supposed to be linted, in this case `utils.i18n/label`, which happens to live in the `utils` namespace.

This way, the `config.edn` could look like this in the future:
```
{:hooks {:analyze-call {utils.i18n/label utils.i18n/label
                        a.b.c/x          a.b.c/x
                        ...
                        }}
 ...
 }
```

This was namespace change was in part inspired by the same approach in e.g.
- https://github.com/taoensso/encore/blob/master/resources/clj-kondo.exports/taoensso/encore/config.edn and 
- https://github.com/taoensso/encore/blob/master/resources/clj-kondo.exports/taoensso/encore/taoensso/encore.clj

This was NOT supposed to be a utility namespace, but I see the confusion.

I am happy to change this back, as this is the only hook so far, and the above approach can be discussed once more hooks are needed.

Reply:
Yeah, hooks are structured in different ways, it's fine if you want to keep it. I guess we have so little usage of hooks at the moment we don't need to overthink.